### PR TITLE
Updated ShimmerEVM Testnet links

### PR DESCRIPTION
**Pull request by anistark**
# Description of change

Updated ShimmerEVM Testnet links

## Type of change

Choose a type of change, and delete any options that are not relevant.

- Documentation Fix

## Change checklist

Tick the boxes that are relevant to your changes, and delete any items that are not.

- [x] I have followed the [contribution guidelines](https://wiki.iota.org/smart-contracts/contribute) for this project
- [x] I have performed a self-review of my own code
- [x] I have selected the `develop` branch as the target branch
- [x] I have made corresponding changes to the documentation

### CODE REVIEW

**Review by jorgemmsilva — APPROVED** [review]

File: documentation/docs/guide/chains_and_nodes/testnet.md [hunk 1]
```diff
@@ -90,7 +90,7 @@ The other values (network name and currency symbol) can be whatever value you li
 
 :::
 
-We have a faucet for you to use directly with your EVM address which can be found on https://deposit.sc.testnet.shimmer.network/
-We also have a withdrawal interface to get any native assets deposited to a EVM chain back into your L1 address: https://withdraw.sc.testnet.shimmer.network/
+We have a faucet for you to use directly with your EVM address which can be found on https://toolkit.sc.testnet.shimmer.network/
+We also have a withdrawal interface to get any native assets deposited to a EVM chain back into your L1 address on the same.
```
Comment (jorgemmsilva) [inline documentation/docs/guide/chains_and_nodes/testnet.md]: On the same?

Comment (anistark) [reply documentation/docs/guide/chains_and_nodes/testnet.md]: We've the withdraw and deposit both in same link now. So, meant like that. Perhaps poor choice of words?

Comment (jorgemmsilva) [reply documentation/docs/guide/chains_and_nodes/testnet.md]: I guess it should say "on the same link" :smile:

Comment (anistark) [reply documentation/docs/guide/chains_and_nodes/testnet.md]: Updated.

**Pull request merged**